src/ResourceManager/ApplicationInsights/Commands.ApplicationInsights/ApplicationInsights/SetApplicationInsightsContinuousExport.cs

```diff
@@ -0,0 +1,169 @@
+            HelpMessage = "Resource Group Name.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Position = 1,
+            Mandatory = true,
+            ParameterSetName = ComponentNameParameterSet,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Application Insights Component Name.")]
+        [Alias(ApplicationInsightsComponentNameAlias, ComponentNameAlias)]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(
+            Position = 2,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Application Insights Continuous Export Id.")]
+        [ValidateNotNullOrEmpty]
+        public string ExportId { get; set; }
+
+        [Parameter(
+            Position = 3,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Document types that need exported.")]        
+        [ValidateSet(ApplicationInsightsBaseCmdlet.DocumentTypes.Requests,
+            ApplicationInsightsBaseCmdlet.DocumentTypes.Exceptions,
+            ApplicationInsightsBaseCmdlet.DocumentTypes.Event,
+            ApplicationInsightsBaseCmdlet.DocumentTypes.Messages,
+            ApplicationInsightsBaseCmdlet.DocumentTypes.Metrics,
+            ApplicationInsightsBaseCmdlet.DocumentTypes.PageViewPerformance,
+            ApplicationInsightsBaseCmdlet.DocumentTypes.PageViews,
+            ApplicationInsightsBaseCmdlet.DocumentTypes.RemoteDependency,
+            ApplicationInsightsBaseCmdlet.DocumentTypes.Availability,
+            ApplicationInsightsBaseCmdlet.DocumentTypes.PerformanceCounters,
+            IgnoreCase = true)]
+        public string[] DocumentType { get; set; }
+
+        [Parameter(
+            Position = 4,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Destination Storage Account Id.")]
+        [ValidateNotNullOrEmpty]
+        public string StorageAccountId { get; set; }
+
+        [Parameter(
+            Position = 5,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Destination Storage Location Id.")]
+        [ValidateNotNullOrEmpty]
+        public string StorageLocationId { get; set; }
+
+        [Parameter(
+            Position = 6,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Destination Storage SAS uri.")]
+        [ValidateNotNullOrEmpty]
+        public string StorageSASUri { get; set; }
+
+        [Parameter(
+            Position = 7,
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Enable continuous export or not.")]
+        [PSDefaultValue(Value = true)]
+        public bool IsEnabled
```

Comment:
@frankguodongchen the use of `bool` parameters is heavily discouraged in PowerShell - we suggest introducing two new parameters, `Enabled` and `Disabled`, that are disjoint `SwitchParameters`. This way, the user will have to explicitly state if they want continuous export enabled or disabled.

Reply:
I am adding Switch parameter DisableConfiguration for this to represent this bool. if it present, it will disabled, otherwise, it will take it as enabled.